src/pages/ProcessMoneyRequestHoldPage.tsx

```diff
@@ -15,7 +15,7 @@ function ProcessMoneyRequestHoldPage() {
     const {translate} = useLocalize();
 
     const onConfirm = useCallback(() => {
-        IOU.setShownHoldUseExplanation();
+        IOU.dismissHoldUseExplanation();
         Navigation.goBack();
     }, []);
 
```

Comment:
In wide screen we handle `onConfirm` here, but I think we may miss handling on modal dismiss

Reply:
It is handled over here and then passed to `OnConfirm` and `OnClose` of the modal:

<img width="453" alt="Screenshot 2024-06-06 at 10 16 11" src="https://github.com/Expensify/App/assets/88325488/a16fc7ed-63a4-4a4b-964c-51a4c4fe6971">

Reply:
@cdOut I think the code you mentioned is on small screen only, but wide screen only handle `onConfirm`, and not handle `OnClose`

 ![Screenshot 2024-06-06 at 8 15 02 PM](https://github.com/Expensify/App/assets/41129870/69b85948-f67a-42b0-8e2d-6b11582ef07e)

Reply:
In the screenshot you provided you can see an override for both the `onClose` and `onConfirm`, I am a little confused about what you exactly mean here? In my opinion all the closing / confirming conditions for both the widescreen and the mobile page / modal are met.

Reply:
@cdOut The code in my screenshot is for small screen, see line 263 on my screenshot `{isSmallScreenWidth && `

Reply:
Ah you're correct, sorry for the confusion! I'll add a handle for this case too.

Reply:
We handle the education modal in different places for wide screen we use `ProcessMoneyRequestHoldPage`, and for small screen we use `ProcessMoneyRequestHoldMenu`

Reply:
I am aware, I wrote the Educational Intersitial for the hold feature 😅. I’ll include this case in the full screen page.